crates/sui-protocol-config/src/lib.rs

```diff
@@ -3254,7 +3283,19 @@ impl ProtocolConfig {
                         cfg.feature_flags.passkey_auth = true;
                     }
                 }
-                76 => {}
+                76 => {
+                    cfg.feature_flags.move_native_context = false;
```

Comment:
Why value 30?

Reply:
as I replied to @tnowacki who asked the same question ;) ...
out of the blue. Until we hook up the code (next PR) there is no way to know what they should be exactly.
I just needed a number and 30 sounded a decent enough number for what the instructions and the push/pop did in the original code?
truth is in the later PR we will see those numbers are not quite good yet

Reply:
see answer to @tnowacki 's same comment